README.md

```diff
@@ -27,6 +27,7 @@ This bundle contains support for 58 different providers:
 * 37signals,
 * Amazon,
 * Asana,
+* Apple
```

Comment:
This is an alphabetically ordered list:
```suggestion
* Apple,
* Asana,
```
(you also forgot the `,`)

Reply:
Fixed